src/data/helper.js

```diff
@@ -23,12 +23,25 @@ export class AuthenticationError extends Error {
     }
 }
 
+export class ForbiddenError extends Error {
+    constructor(statusCode, body) {
+        super('You cannot perform this action');
+        this.name = 'ForbiddenError';
+        this.statusCode = statusCode;
+        this.body = body;
+    }
+}
+
 export function throwIfNotSuccess(response) {
     if (!response.ok) {
         if (response.status === 401) {
             return new Promise((resolve, reject) => {
                 response.json().then(body => reject(new AuthenticationError(response.status, body)));
             });
+        } else if (response.status === 403) {
+            return new Promise((resolve, reject) => {
+                response.json().then(body => reject(new ForbiddenError(response.status, body)));
+            });
```

Comment:
We have actually used status code `403` in the scenario where the toggle name is already in use. 

This change ends up changing the user feedback to:
![image](https://user-images.githubusercontent.com/158948/50239154-b4e52b80-03c1-11e9-91b3-612f5cbeda65.png)

Previously it was:
![image](https://user-images.githubusercontent.com/158948/50239265-fece1180-03c1-11e9-8bb8-2f6552eb7be4.png)

I think the bug here is that `unleash-server` uses 403 and not `400` for the name-already exists scenario. 

Do you agree? If yes **I will** be happy to change unleash-server to use correct status-code.

Reply:
If the permissions system is going to be merged, 403 should be solely reserved for "Forbidden" errors, so for when the client is missing permissions.
Is there any reason you wouldn't be able to change this with a simple number change?

Reply:
nope. I will head on fixing this ahead of the other PR on unleash-server.

Reply:
resolved: https://github.com/Unleash/unleash/commit/59ef1d356b9300c276086800e1acbb16df10303c